src/composable/types/Twap.ts

```diff
@@ -127,14 +128,8 @@ export class Twap extends ConditionalOrder<TwapData, TwapDataParams> {
     // First, verify that the handler is the TWAP address
     if (handler !== TWAP_ADDRESS) throw new Error(`InvalidHandler: Expected: ${TWAP_ADDRESS}, provided: ${handler}`)
 
-    // Second, verify that the order params are logically valid
-    Twap.isValid(staticInput)
-
     // Third, construct the base class using transformed parameters
     super({ handler: TWAP_ADDRESS, salt, staticInput, hasOffChainInput })
-
-    // Finally, verify that the transformed data is ABI-encodable
-    if (!isValidAbi(TWAP_DATA_ABI, [this.staticInput])) throw new Error('InvalidData')
```

Comment:
If we can instantiate a concrete order that isn't valid at all times, I'd suggest that in `Multiplexer` within the `add` function that validation takes place to avoid `SDK` users inserting invalid orders (ideally, these should be caught well before the merkle tree attempts to generate itself).

Reply:
Great point!

Reply:
Added the check, and a test for it (great catch):

<img width="1068" alt="image" src="https://github.com/cowprotocol/cow-sdk/assets/2352112/3975feb0-803a-482d-be3c-e1a4c8b68206">